php/config-spec.php

```diff
@@ -8,8 +8,8 @@
 	),
 
 	'ssh' => array(
-		'runtime' => '=<ssh>',
-		'file' => '<ssh>',
+		'runtime' => '=<user>[:<pass>]@<host>[:<port>]<path>',
+		'file' => '<user>[:<pass>]@<host>[:<port>]<path>',
```

Comment:
`<user>` is optional, as is `[<path>]`. Really, only `<host>` is required.

Is `[:<pass>]` actually supported?